package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "resin-bundle-resolve",
+  "version": "0.0.1",
+  "description": "Resolve resin-bundles into a docker daemon format",
+  "main": "lib/index.js",
+  "types": "src/index.ts",
+  "scripts": {
+    "start": "",
+    "build": "gulp",
+    "pretest": "tsc tests/tests.ts",
```

Comment:
Ah, that's interesting. I think you should be able to use `include` instead of rootDir. RootDir will get calculated automatically from the files used, so if your include is everything in the src folder, then rootDir is `src/` by default, and if you then explicitly ask it to compile the tests (by letting Mocha do so), it should override that, calculate the rootDir correctly as the root of the whole project, and you'll be all good.

Or maybe it won't, I haven't tested this. Have a go, if you get stuck throw a broken version on a branch and I'll help work out how to fix it :smile: